core/crypto/crypto.go

```diff
@@ -49,13 +49,13 @@ type Client interface {
 	Node
 
 	// NewChaincodeDeployTransaction is used to deploy chaincode.
-	NewChaincodeDeployTransaction(chaincodeDeploymentSpec *obc.ChaincodeDeploymentSpec, uuid string) (*obc.Transaction, error)
+	NewChaincodeDeployTransaction(chaincodeDeploymentSpec *obc.ChaincodeDeploymentSpec, attributes []string, uuid string) (*obc.Transaction, error)
```

Comment:
@adecaro transaction can be deployed, invoked and queried in the old way too, in other words transaction can be generated without use values preserving retro-compatibility.